db/schema.rb

```diff
@@ -0,0 +1,88 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 20_220_517_170_114) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'foods', force: :cascade do |t|
+    t.string 'name'
+    t.string 'measurement_unit'
+    t.integer 'price'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'user_id', null: false
+    t.index ['user_id'], name: 'index_foods_on_user_id'
+  end
+
+  create_table 'inventories', force: :cascade do |t|
+    t.string 'name'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'user_id', null: false
+    t.index ['user_id'], name: 'index_inventories_on_user_id'
+  end
+
+  create_table 'inventory_foods', force: :cascade do |t|
+    t.integer 'quantity'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'inventory_id', null: false
+    t.bigint 'food_id', null: false
+    t.index ['food_id'], name: 'index_inventory_foods_on_food_id'
+    t.index ['inventory_id'], name: 'index_inventory_foods_on_inventory_id'
+  end
```

Comment:
- [OPTIONAL] Creating `Inventories list` and `Inventory details` and other inventory related pages are additional requirements for your team because you are a group of two people. Those are specifically for teams with 3 members. So feel free to delete all the inventory related tables, views, controllers, models and routes because in your app they are not used by a user anyway.